lsp-bridge-call-hierarchy.el

```diff
@@ -0,0 +1,197 @@
+      (lsp-bridge-call-hierarchy-maybe-preview)
+      ))
+
+(defun lsp-bridge-call-hierarchy-next ()
+  (interactive)
+  (lsp-bridge-call-hierarchy--move 1))
+
+(defun lsp-bridge-call-hierarchy-prev ()
+  (interactive)
+  (lsp-bridge-call-hierarchy--move -1))
+
+(defun lsp-bridge-call-hierarchy-quit ()
+  (interactive)
+  (posframe-delete "*lsp-bridge-call-hierarchy*")
+  (select-frame-set-input-focus lsp-bridge-call-hierarchy--emacs-frame)
+
+  (dolist (buffer lsp-bridge-call-hierarchy--temp-buffers)
+    (kill-buffer buffer)))
+
+(defun lsp-bridge-call-hierarchy-show ()
+  (let* ((call  (nth lsp-bridge-call-hierarchy--index lsp-bridge-call-hierarchy--popup-response))
+         (path (plist-get call :path))
+         (range-start (plist-get (plist-get call :range) :start))
+         (range-end (plist-get (plist-get call :range) :end))
+         (sline (plist-get range-start :line))
+         (scharacter (plist-get range-start :character))
+         (eline (plist-get range-end :line))
+         (echaracter (plist-get range-end :character))
+         (found-buffer))
+
+    (if (dolist (buffer (buffer-list) found-buffer)
+          (when (string-equal (buffer-file-name buffer) path)
+            (setq found-buffer buffer)))
+        (find-file path)
+      (add-to-list 'lsp-bridge-call-hierarchy--temp-buffers (find-file path)))
+
+    (goto-char (point-min))
+    (forward-line sline)
+    (goto-char (+ (point-at-bol) scharacter))
+    (if (equal sline eline)
+        (pulse-momentary-highlight-region (point)
+                                          (+ (point) (- echaracter scharacter)))
+      (pulse-momentary-highlight-region (point)
+                                        (save-excursion
+                                          (forward-line (- eline sline))
+                                          (goto-char (+ (point-at-bol) echaracter))
+                                          (point))))))
+
+(defun lsp-bridge-call-hierarchy-select ()
+  (interactive)
+  (lsp-bridge-call-hierarchy-show)
+  (lsp-bridge-call-hierarchy-quit))
+
+(defvar lsp-bridge-call-hierarchy-mode-map
+  (let ((map (make-sparse-keymap)))
+    (define-key map (kbd "j") 'lsp-bridge-call-hierarchy-next)
+    (define-key map (kbd "k") 'lsp-bridge-call-hierarchy-prev)
+    (define-key map (kbd "RET") 'lsp-bridge-call-hierarchy-select)
+    (define-key map (kbd "ESC") 'lsp-bridge-call-hierarchy-quit)
+    (define-key map (kbd "q") 'lsp-bridge-call-hierarchy-quit)
+  map))
+
+(define-derived-mode lsp-bridge-call-hierarchy-mode text-mode "call hierarchy"
+  ;; Kill all local variables.
+  (kill-all-local-variables)
+  ;; Switch new mode.
+  (setq major-mode 'lsp-bridge-call-hierarchy-mode)
+  (setq mode-name "call hierarchy")
+  (setq-local cursor-type nil)
+  (goto-char (1+ (point-min)))
+  (setq lsp-bridge-call-hierarchy--overlay (make-overlay (line-beginning-position)  (line-end-position)))
+  (overlay-put lsp-bridge-call-hierarchy--overlay 'face 'highlight)
```

Comment:
Execute command `lsp-bridge-outgoing-call-hierarchy` and press `j` in frame, font-size will become bigger, maybe we should use customize face instead `highlight` face to fix this issue?